src/parsec_service/source_code_structure.md

```diff
@@ -86,12 +86,12 @@ underlying security hardware to determine if the request has been correctly auth
 more detailed description of authentication and application identity, see the [**API
 overview**](../parsec_client/api_overview.md).
 
-The `key_id_managers` module offers functionality for the providers for persisting mappings between
-key names and provider-specific key IDs. The key ID managers close the gap between the API, which
-allows keys to be addressed by a UTF-8 name, and the providers which have specific requirements for
-key handles (e.g. Mbed Crypto uses a 32 bit value for handles of persistent keys). Using a key ID
-manager is only required for persistent keys and the only current implementation stores the mappings
-on disk.
+The `key_info_managers` module offers functionality for the providers for persisting mappings
+between key names and provider-specific key information. The key info managers close the gap between
+the API, which allows keys to be addressed by a UTF-8 name, and the providers which have specific
+requirements for key handles (e.g. Mbed Crypto uses a 32 bit value for handles of persistent keys).
+Using a key info manager is only required for persistent keys and the only current implementation
+stores the mappings on disk.
```

Comment:
While this is certainly useful to know, I'm not sure it should be here. It feels like we have a lot of information scattered over various documents and it'll be difficult to make sure we keep everything updated - e.g. for this phrase, I'm not sure we'll remember to update it when we'll have a different key info manager, since this page seems only vaguely related to what managers we actually have. We can create an issue to track this, potentially (re)moving doc parts that aren't in the most obvious place.

Reply:
True, will create one.